crates/prover-clients/atlantic-service/src/client.rs

```diff
@@ -85,8 +85,7 @@ impl AtlanticClient {
             )
             .send()
             .await
-            .map_err(AtlanticError::AddJobFailure)
-            .expect("Failed to add job");
+            .map_err(AtlanticError::AddJobFailure)?;
```

Comment:
Same, can add `wrap_err` here

WDYT on adding `wrap_err` throughout the code @Mohiiit ?

Reply:
I don't think it makes sense here, because wrap_err is used to provide extra context to the error, in here the only context is that the add job failure happened which is already handled by the AtlanticError here